snapcraft.yaml

```diff
@@ -28,6 +28,11 @@ plugs:
     interface: system-files
     write:
       - /etc/sysctl.d/50-aws-gadget.conf
+  # workaround because the nf_conntrack module gets loaded after systemd-sysctl.service runs (LP:#2125739)
+  proc-sys-net-netfilter-nf_conntrack_max:
```

Comment:
Underscores are not allowed in dictionary keys by snapd, this should be `proc-sys-net-netfilter-nf-conntrack-max` to avoid being ignored by snapd.

Reply:
fixed